Makefile

```diff
@@ -18,7 +18,7 @@ $(if $(findstring $(space),$(TOPDIR)),$(error ERROR: The path to the OpenWrt dir
 
 world:
 
-DISTRO_PKG_CONFIG:=$(shell command -pv pkg-config | grep -E '\/usr' | head -n 1)
```

Comment:
This seem to be the only place which behaves differently than `which`. Are you sure the other changes are required?

Reply:
Most command invocations include "-p", which resets the PATH to a "standard path". That's not the behavior of which and this is breaking my build. There is one correct change in the original commit, which isn't worth keeping. Also given that I am going to retain the which dependency the other changes become somewhat pointless.

Reply:
> and this is breaking my build

Ok, how can I reproduce the breakage?